etc/kayobe/ansible/pull-retag-push.yml

```diff
@@ -80,24 +80,24 @@
     - name: Pull container images (may take a long time)
       become: true
       vars:
-        remote_image: "{% if docker_pull_registry != '' %}{{ docker_pull_registry }}/{% endif %}{{ docker_pull_namespace }}/{{ item }}:{{ docker_pull_tag }}"
+        remote_image: "{% if docker_pull_registry != '' %}{{ docker_pull_registry }}/{% endif %}{{ docker_pull_namespace }}/{{ item }}:{{ docker_pull_tag }}-{{ kolla_base_distro }}-{{ ansible_facts.distribution_release }}"
```

Comment:
I don't think ansible_facts.distribution_release is always correct here. For example, on CentOS stream it is `stream8`.

Reply:
Would ansible_distribution_release be better?

Reply:
That's just a different way of accessing the same thing. Here's how kolla-ansible does it:

https://opendev.org/openstack/kolla-ansible/src/commit/a396284a83e7624a951190b36fbad79812f1937d/ansible/group_vars/all.yml#L1196